internal/api/block_documents.go

```diff
@@ -42,15 +42,21 @@ type BlockDocumentUpdate struct {
 	MergeExistingData bool                   `json:"merge_existing_data"`
 }
 
-// BlockDocumentAccessReplace is the "update" request payload
+// BlockDocumentAccessUpsert is the create/update request payload
 // to modify a block document's current access control levels,
 // meaning it contains the list of actors/teams + their respective access
 // to a given block document.
-type BlockDocumentAccessReplace struct {
-	ManageActorIDs []AccessActorID `json:"manage_actor_ids"`
-	ViewActorIDs   []AccessActorID `json:"view_actor_ids"`
-	ManageTeamIDs  []uuid.UUID     `json:"manage_team_ids"`
-	ViewTeamIDs    []uuid.UUID     `json:"view_team_ids"`
+type BlockDocumentAccessUpsert struct {
+	AccessControl struct {
+		ManageActorIDs []string `json:"manage_actor_ids"`
+		ViewActorIDs   []string `json:"view_actor_ids"`
+		ManageTeamIDs  []string `json:"manage_team_ids"`
+		ViewTeamIDs    []string `json:"view_team_ids"`
+		// ManageActorIDs []AccessActorID `json:"manage_actor_ids"`
+		// ViewActorIDs   []AccessActorID `json:"view_actor_ids"`
+		// ManageTeamIDs  []uuid.UUID     `json:"manage_team_ids"`
+		// ViewTeamIDs    []uuid.UUID     `json:"view_team_ids"`
```

Comment:
Did you want to leave these around in case we change how we convert the values?

Reply:
heh - i actually ended up changing my approach, i think i was getting a little too cute with the custom types and we really only needed string types for the ID / `*`.  ill clean these out